static/src/javascripts/projects/common/modules/save-for-later.js

```diff
@@ -82,6 +84,14 @@ define([
         return { prop74: prefix + 'ContainerSave:' + contentId };
     };
 
+    SaveForLater.prototype.conditionalInit = function(){
+        if (!liveBlog.notificationsCondition()) {
+            var $saveForLaterContainer = $('.meta__save-for-later');
+            $saveForLaterContainer.show();
```

Comment:
This has been removed; `init` shows the containers anyway, so this is not needed. Not sure why github keeps these comments laying around now.